rust/processor/src/models/object_models/v2_objects.rs

```diff
@@ -25,6 +34,29 @@ use std::collections::HashMap;
 // PK of current_objects, i.e. object_address
 pub type CurrentObjectPK = String;
 
+/// Tracks all object related metadata in a hashmap for quick access (keyed on address of the object)
+pub type ObjectAggregatedDataMapping = HashMap<CurrentObjectPK, ObjectAggregatedData>;
+
+/// Index of the event so that we can write its inverse to the db as primary key (to avoid collisiona)
+pub type EventIndex = i64;
+
+/// This contains metadata for the object
```

Comment:
Let's specify that it's only including fungible assets and token metadata currently.